contracts/zero-ex/contracts/src/transformers/bridges/mixins/MixinCompound.sol

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+
+  Copyright 2021 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.6.5;
+pragma experimental ABIEncoderV2;
+
+import "@0x/contracts-erc20/contracts/src/v06/LibERC20TokenV06.sol";
+import "@0x/contracts-erc20/contracts/src/v06/IERC20TokenV06.sol";
+import "@0x/contracts-erc20/contracts/src/v06/IEtherTokenV06.sol";
+import "@0x/contracts-utils/contracts/src/v06/LibSafeMathV06.sol";
+
+
+// Minimal CToken/CEther interfaces
+interface ICToken {
+    function mint(uint mintAmountInUnderlying) external returns (uint);
+    function redeem(uint redeemTokensInCtokens) external returns (uint);
+}
+interface ICEther {
+  function mint() payable external;
+  function redeem(uint redeemTokens) external returns (uint);
+}
```

Comment:
nit: there's a mix of 2-space and 4-space indents in this file 😛

Reply:
lolol, I guess my editor code formatting is entirely broken for solidity 🙈  Thanks for noticing!